Makefile

```diff
@@ -9,7 +9,8 @@ fix: vendor/autoload.php ## Fixes code style issues with phpcbf
 	vendor/bin/phpcbf
 
 sniff: vendor/autoload.php ## Detects code style issues with phpcs
-	vendor/bin/phpcs
+	mkdir -p .build/php-codesniffer
+	vendor/bin/phpcs --cache=.build/php-codesniffer/cache.json
```

Comment:
Cannot use a cache file when running `phpcbf` - that's why the configuration is passed when invoking the command.